lightning/src/ln/channelmonitor.rs

```diff
@@ -397,6 +396,20 @@ enum InputMaterial {
 	}
 }
 
+#[cfg(test)]
+pub const ACCEPTED_HTLC_SCRIPT_WEIGHT: usize = 138; //Here we have a diff due to HTLC CLTV expiry being < 2^15 in test
```

Comment:
Hmmm, can we just drop these constants from channelmonitor and replace them with an fn weight_to_type(weight: usize) -> type enum.

Reply:
You mean type_to_weight  ? As these constants are used as type assertions based on a known weight